pkg/segment/utils/segconsts.go

```diff
@@ -111,6 +111,8 @@ const MS_IN_MIN = 60_000     // 60 * 1000
 const MS_IN_HOUR = 3_600_000 // 60 * 60 * 1000
 const MS_IN_DAY = 86_400_000 // 24 * 60 * 60 * 1000
 
+const MAX_LIST_SIZE = 100
```

Comment:
nit: just from this it's not obvious that this is for the splunk `list` command (as opposed to a normal list in Go), and it's not obvious why it's `100`.

You should rename it to something like `MAX_SPL_LIST_SIZE` (for the Search Processing Language that splunk has), and add a comment about why it's 100 (perhaps linking to splunk documentation)

Reply:
switched to MAX_SPL_LIST_SIZE , and added some description to where the constant comes from.